portfolio/src/main/webapp/static/scripts/comments.js

```diff
@@ -50,7 +50,7 @@ const loadComments = async (recentPostKeyObject) => {
         })
     );
   let text = await response.text();
```

Comment:
Could define an error handle for the http response to avoid having to log on the console here